Wireframe/index.html

```diff
@@ -8,26 +9,45 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>Wireframe to web code</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        Welcome to my ITP Onboarding Module Wirefare to web code.
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="placeholder.svg" alt="readmeimg" width="100%"/>
```

Comment:
You're very close now with the alt text. Can you make the alt text into words that are meaninful? If I'm using a screen reader, I may not know what a 'readmeimg' is.